### ISSUE TITLE
Preloading models is totally unrelated to the sv_pure bypass

### ISSUE THREAD

**Issue by ellie12134** [ValveSoftware/Source-1-Games#5173]
In another Issue report I received a reply that I thought it was necessary to bring up but they blocked me and I thought it was important to clarify

"as well while the anti invis issue has been pointed out to not exactly work right, there are some actual exploits that can be done with this like putting a particle effect on the Spy model so it follows them around."

_Originally posted by @TF2CutContent in https://github.com/ValveSoftware/Source-1-Games/issues/5170#issuecomment-1702984253_

No, the thing they mention about attaching particles to a model is totally unrelated to the sv_pure exploit, the Game preloads models anyways, regardless of sv_pure 1 and 2 , it doesn't matter the config the game will preload models since it's not a thing handled by the sv_pure parameters at all , they are mixing up different things and I think it's worth pointing it's not the smartest thing to directly mention exploit methods so publicly since they are helping spreading awareness on how to perform them, these kind of reports should be threated in a private manner with Valve devs

**Comment by TF2CutContent** [comment]
Duplicate of #5170, is wasting @kisak-valve's time

**Comment by ellie12134** [comment]
> Duplicate of #5170, is wasting @kisak-valve's time

It's only a duplicate because you blocked me. i would happily correct you in the proper post but I am unable to, ane it's important to clarify misconceptions

**Comment by TF2CutContent** [comment]
> > Duplicate of #5170, is wasting @kisak-valve's time
> 
> It's only a duplicate because you blocked me. i would happily correct you in the proper post but I am unable to, and it's important to 

> > Duplicate of #5170, is wasting @kisak-valve's time
> 
> It's only a duplicate because you blocked me. i would happily correct you in the proper post but I am unable to, and it's important to clarify misconceptions

Preloading is indeed part of the bypass hence why there's preloading maps made specifically to not be actually playable, but to be used for the bypass on Gamebanana to preload a bunch of models with VScript. Come on, at least try to make an effort to not sound clearly biased.

**Comment by ellie12134** [comment]
> > > Duplicate of #5170, is wasting @kisak-valve's time
> > 
> > 
> > It's only a duplicate because you blocked me. i would happily correct you in the proper post but I am unable to, and it's important to
> 
> > > Duplicate of #5170, is wasting @kisak-valve's time
> > 
> > 
> > It's only a duplicate because you blocked me. i would happily correct you in the proper post but I am unable to, and it's important to clarify misconceptions
> 
> Preloading is indeed part of the bypass hence why there's preloading maps made specifically to not be actually playable, but to be used for the bypass on Gamebanana to preload a bunch of models with VScript. Come on, at least try to make an effort to not sound clearly biased.

The only part of the bypass that is related to sv_pure is the specific hud materials folders, since they allow the usage of custom VMTs and VTFs with custom models, but custom models exploit wouldnt use said materials folders since it's as easy as attaching a particle to it, but then again that doesn't fall under the pure exploit but instead the preloading exploit

**Comment by TF2CutContent** [comment]
If you had this much to say, then you should've said it instead of leaving a pointless message on the actual issue report, getting yourself blocked by me for wasting my time, and wasting kisak's time having to close this.

**Comment by ellie12134** [comment]
> 

I didnt had to mention the preload bypass until you yourself mentioned which by the time I was already blocked, you're being inconsistent in the discussion and you havent provided enough arguments to prove what I am saying as false and yet you wont allow for the debate to carry on

**Comment by kisak-valve** [comment]
@TF2CutContent, it sounds like you're the cause of the discussion fragmentation here. You're the one who has chosen to force @ellie12134 out of the other issue report for a relatively minor first indiscretion, and for whatever reason Github has let you do it.

You should have a better reason than a single neutral comment has mildly inconvenienced you by existing. I'm going to frown on this choice, but take no immediate action.

@ellie12134, for the sake of continuity, I've transferred this issue report to https://github.com/ValveSoftware/Source-1-Games/issues/5170#issuecomment-1703032817.
